gcloud/datastore/_implicit_environ.py

```diff
@@ -126,7 +122,8 @@ def get_connection():
     :rtype: :class:`gcloud.datastore.connection.Connection`
     :returns: A connection defined with the proper credentials.
     """
-    return get_scoped_connection(Connection, SCOPE)
+    credentials = get_credentials()
```

Comment:
In some sense, we could beef up `Connection.__init__` and ditch this method all together.